app/controllers/users_controller.rb

```diff
@@ -1,9 +1,10 @@
 class UsersController < ApplicationController
   def index
-    @users = User.all
+    @users_all = User.all
   end
 
   def show
-    @users = User.find(params[:id])
+    @users_find = User.find(params[:id])
+    @post = Post.find(params[:id])
   end
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/231342062-c135faf2-6747-45df-8495-4ccc32e7d9a5.png)

- [ ] When I clicked on that user which has no posts then i saw this error which you can see in the image above :up: Kindly fix this error to make your project work as expected :+1: